molecule/default/tasks/gc.yml

```diff
@@ -46,6 +46,9 @@
         wait: yes
         wait_timeout: 100
       register: job
+      until: job.resources[0].status.phase == "Running"
```

Comment:
There are a couple other places in this file that we should probably do the same thing for.